subprojects/docs/src/docs/userguide/dependencyTypes.adoc

```diff
@@ -0,0 +1,96 @@
+[[dependency_types]]
+== Dependency types
```

Comment:
All other page titles use Title Case, so I think we should use that to keep that consistent.

Reply:
I believe I recently pinged you about how to format titles on Slack. I take it we'll want to use Title Case?